### PR TITLE
feat: Implement image sequence publisher

### PR DESCRIPTION
**Pull request by henriknorin-ftrack**
Resolves : 

* CLICKUP-3vta7gf

This PR has been tested on :

- [X] Windows.
- [ ] MacOs.
- [ ] Linux.

## Changes

Implement image sequence publisher, with reviewable component.

## Test

_Note: needs to be tested with corresponding branch on definition and unreal._

Setup sequencer with an animated camera and run image sequence publisher, it should render and publish EXRs + AVI reviewable to frack as expected.

### CODE REVIEW

File: resource/plugins/unreal/python/publisher/validators/unreal_sequence_publisher_validator.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+from ftrack_connect_pipeline_unreal import plugin
+
+# import maya.cmds as cmds
```
Comment (lluisCM) [inline resource/plugins/unreal/python/publisher/validators/unreal_sequence_publisher_validator.py]: remove this

**Review by lluisCM — CHANGES_REQUESTED** [review]
LGTM, but please add some docstring on the files. Thanks.

File: resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+):
+    plugin_name = 'unreal_sequence_publisher_collector'
+
+    def select(self, context_data=None, data=None, options=None):
+        '''Select all the sequences in the given plugin *options*'''
+        selected_items = options.get('selected_items', [])
+        return selected_items
+
+    def fetch(self, context_data=None, data=None, options=None):
+        '''Fetch all cameras from the scene'''
```
Comment (lluisCM) [inline resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py]: Sequence not cameras?

File: resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        for object in collected_objects:
+            data = {'value': object}
+            if object == seq_name_sel:
+                data['default'] = True
+            result.append(data)
+        return result
+
+    def run(self, context_data=None, data=None, options=None):
+        '''Return the long name of the camera from the plugin *options*'''
```
Comment (lluisCM) [inline resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py]: Sequence not cameras?

**Review by lluisCM — APPROVED** [review]
Please fix comments before merge

File: resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py (hunk 2 above)
Comment (henriknorin-ftrack) [reply resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py]: Fixing

File: resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py (hunk 3 above)
Comment (henriknorin-ftrack) [reply resource/plugins/unreal/python/publisher/collectors/unreal_sequence_publisher_collector.py]: Fixing

**Pull request merged**